content/dev/code/notifiers.md

```diff
@@ -3,420 +3,243 @@ title: Notifiers and Observers - About
 
-The observer/notifier system is an implementation of the ["pub-sub" pattern](https://en.wikipedia.org/wiki/Publish–subscribe_pattern).  It was introduced to give developers deep access to core operation, without the need to touch any core files at all. Although it was written for an object-oriented code base, it can be used with procedural code as well.
+The observer/notifier system is an implementation of the ["pub-sub" pattern](https://en.wikipedia.org/wiki/Publish–subscribe_pattern)  that was introduced to give developers deep access to core operation without the need to touch any core files at all. Although the implementation was written for an object-oriented code base, it can alse be used with procedural code.
 
-### Extending All Classes
+At a high level, a developer identifies an _event_ that is "interesting", e.g. a customer has just successfully logged in, and registers to be notified when that event occurs.  If/when that event occurs, the `base` class receives control and looks to see if any registrations exist for the event.  If so, all registered observer-classes are called to provide their custom actions.
 
-In order to use notifiers in a class, you will need to make that class extend the `base` class. eg:
+Here are some 'quick links' to various sections of this documentation:
 
-<pre>  class currencies extends base {
-</pre>
+1. [Issuing Event Notifications](issuing-event-notifications).  Identifies the mechanism used by _event issuers_ to issue a notification.
 
+2. [Observing Notifications](observing-notifications). Identifies the mechanisms used by _event observers_ to perform their customizations.
 
-### Notifiers: Big Brother is watching
+3. [Loading Your Observer Class](loading-your-observer-class).  Identifies how to load your observer-class so that it can begin its observations.
 
-The point of the observer/notifier system is to permit developers to write code that listens for certain events to happen, and then when they do, have their own code executed.
+4. [Advanced Topics](advanced-topics). This section identifies additional methods that can be used to auto-load an observer-class and create event-specific event-handlers.
 
-Events are simply mnemonic strings.  You can see a 
-[list of notifiers](/dev/code/notifiers_list) for reference.
+    a. [Choosing When to Load an Observer](choosing-when-to-load-an-observer).
+    b. [Auto-loaded Observers](auto-loaded-observers).
+    c. [Event-Specific Update Methods](event-specific-update-methods).
 
-Events are triggered by calling `$this->notify` in any class that `extends base`:
+5. [Additional Information](additional-information).  This section has references to additional documentation on the observer/notifier system.
 
-<pre> $this->notify('EVENT_NAME');
-</pre>
 
-Example: In the shopping cart class after an item has been added to the cart this event is triggered:
+## Issuing Event Notifications
```

Comment:
I feel like the word "issue" doesn't fully capture what's happening. Perhaps "trigger" would be more clear, and allow easier wording later about "triggering and issuer" which could be replaced with "triggered".

Reply:
Agreed.  Just remember that there are intra-page links at the top of the document, so those would need changing as well if the level-2-heading text is changed.